api/models/user_custom_fields.py

```diff
@@ -8,14 +8,38 @@ class UserCustomFieldsQuery(BaseQuery):
     def get_by_user_phone(self, phone):
         return self.filter(UserCustomFields.user_phone == phone).all()
 
+    def mark_user_groups_as_inactive(self, phone):
```

Comment:
Why we are using the `user_custom_fields` model for updating `user_groups`?

Reply:
Missed updating the name of the function. I'll correct it.